utils/spacewalk-common-channels.ini

```diff
@@ -1218,6 +1275,12 @@ repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/
 name     = Uyuni Client Tools for %(base_channel_name)s (Development)
 archs    = x86_64, aarch64
 base_channels = opensuse_micro5_3-%(arch)s
+
+# This is expected. openSUSE Leap 15.0 client tools are valid for all openSUSE Leap 15.X releases
+[opensuse_leap15_5-uyuni-client-devel]
+name     = Uyuni Client Tools for %(base_channel_name)s (Development)
+archs    = x86_64, aarch64
+base_channels = opensuse_leap15_5-%(arch)s
```

Comment:
Nope, this conflict was wrongly fixed.

See above, you broke my opensuse_micro5_3-uyuni-client-devel definition.